distribution/springboot_setup/sender/routes/out-bound-lims-route.xml

```diff
@@ -0,0 +1,204 @@
+                    <jsonpath>$.occurrencePeriod.start</jsonpath>
+                </setProperty>
+
+                <setProperty name="patient-id">
+                    <jsonpath>$.subject.identifier.id</jsonpath>
+                </setProperty>
+
+                <!--to retrieve real patient names using fhir-->
+
+                <setProperty name="patient-name">
+                    <jsonpath>$.subject.display</jsonpath>
+                </setProperty>
+
+                <!--to retrieve real requester names using fhir-->
+
+                <setProperty name="requester-first-name">
+                    <jsonpath>$.requester.identifier.value</jsonpath>
+                </setProperty>
+
+                <setProperty name="requester-last-name">
+                    <jsonpath>$.requester.identifier.value</jsonpath>
+                </setProperty>
+
+               <!--setting up authentication to SENAITE-->
+
+                <setBody>
+                    <simple>admin:admin</simple>
+                </setBody>
+                <marshal>
+                    <base64/>
+                </marshal>
+                <convertBodyTo type="java.lang.String"/>
+                <setHeader name="Authorization">
+                    <simple>Basic ${body}</simple>
+                </setHeader>
+                <setHeader name="CamelHttpMethod">
+                    <constant>GET</constant>
+                </setHeader>
+                <setHeader name="Content-Type">
+                    <constant>application/json</constant>
+                </setHeader>
+
+                <!--create client(from ServiceRequest.subject) on SENAITE and save client UID property-->
+
+                <toD uri="http://34.252.234.113:8080/senaite/@@API/senaite/v1/search?portal_type=Client&amp;getName=${exchangeProperty.patient-name}"/>
+                
+                <unmarshal>
+                    <json library="Jackson"/>
+                </unmarshal>
+
+                <choice>
+                    <when>
+                        <jsonpath>$.[?(@.count==1)]</jsonpath>
+                        <setProperty name="client-uid">
+                            <jsonpath>$.items[0].uid</jsonpath>
+                        </setProperty>
+                        <setProperty name="client-storage-path">
+                            <jsonpath>$.items[0].path</jsonpath>
+                        </setProperty>
+                    </when>
+                    <otherwise>
+                        <setBody>
+                            <simple>{"portal_type":"Client","title":"${exchangeProperty.patient-name}","ClientID":"${exchangeProperty.patient-id}","parent_path":"/senaite/clients"}</simple>
+                        </setBody>
+
+                        <setHeader name="CamelHttpMethod">
+                            <constant>POST</constant>
+                        </setHeader>
+                        <toD uri="http://34.252.234.113:8080/senaite/@@API/senaite/v1/create"/>
```

Comment:
Yes for sure @mks-d. I left the url hard-coded for easier testing since I'm currently depending on an older version of the eip. I'll do as done on [line 33](https://github.com/openmrs/openmrs-eip/pull/13/files#diff-58ff57c01537af6124e08c10614a91cd447b7ebc2d36e4a9633c56d65abd5063R33) for the lims base url. Thanks.